go/worker/compute/executor/committee/node.go

```diff
@@ -1384,18 +1495,16 @@ func (n *Node) worker() {
 	// We are initialized.
 	close(n.initCh)
 
-	var runtimeVersion version.Version
 	for {
 		// Check if we are currently processing a batch. In this case, we also
 		// need to select over the result channel.
-		var processingDoneCh chan *processedBatch
-
+		n.processingDoneCh = nil
```

Comment:
Maybe it would be better to keep the variable here and just pass it as an argument to `n.handleProcessedBatch` to avoid anyone using it elsewhere.